### PR TITLE
Add youtube and peertube links to about page

### PR DESCRIPTION
**Pull request by hunicus**
Screenshot from 2023-03-07 05-02-39

### CODE REVIEW

**Review by softsimon — CHANGES_REQUESTED** [review]
tested NACK

The positioning looks off

Screenshot 2023-03-08 at 11 52 13

**Comment by hunicus** [comment]
Switching the PeerTube logo for a BitcoinTV.com logo but I'm seeing some weird issue. Outlines within the shape are showing at smaller sizes and I'm not sure how to fix. 

Reaching out to @pedromvpg for help.

Icon on page (note the outlines):

Screenshot from 2023-03-08 03-15-03

Icon in editor does not (red is transparent):

Screenshot from 2023-03-08 03-04-51

**Comment by hunicus** [comment]
@softsimon ready for review.

Screenshot:

Screenshot from 2023-03-12 03-02-22

Vertical alignment:

Screenshot from 2023-03-12 03-03-46

Intentionally excluded lightning bolt for vertical alignment...looks better this way.

File: frontend/src/app/components/about/about.component.html [hunk 1]
```diff
@@ -383,6 +383,12 @@ <h3 i18n="about.maintainers">Project Maintainers</h3>
...
...7-7.67l-1.12-.94c-6.89-5.78-11.41-10.21-14.23-13.94-3.44-4.56-4.32-8.05-2.67-10.65l.33-.53c1.19-1.5 3.28-2.72 6.23-3.64a63.83 63.83 0 0 1 18.84-2.59 117.38 117.38 0 0 1 21.72 2.27 213.76 213.76 0 0 1 23.35 6c16.76 5.23 34 12.35 47.18 18A88.41 88.41 0 0 1 272 448.86Zm88.28-135.81c-4.47.48-9 1.14-13.38 2a72.64 72.64 0 0 1-12.75 1.13 81.06 81.06 0 0 1-12.73-1 100.53 100.53 0 0 1-12.72-2.91 146.76 146.76 0 0 1-25.3-10.47c-16.86-8.65-33.41-20.17-48.91-31.23l-5.35-3.82c-13.55-9.69-26.34-18.84-38.52-25.82a121.47 121.47 0 0 0-17.73-8.54 55.57 55.57 0 0 0-16.53-3.72h-2a18.9 18.9 0 0 0-7.22 1.16 7.65 7.65 0 0 0-3.94 3.41c-1.06 2-1.22 4.55-.51 7.83a44.86 44.86 0 0 0 3.95 10.28c3.63 7.23 9.11 15.57 14.91 24.22l.47.69 1.18 1.74c6.33 9.41 12.87 19.14 16.79 27.05l.21.44c3.44 7.1 4.35 12 2.78 14.85a5.48 5.48 0 0 1-2.19 2.16 11 11 0 0 1-4 1.17 70.91 70.91 0 0 1-8.5.51 74.12 74.12 0 0 1-14.4-1.45 116.93 116.93 0 0 1-13.89-3.79l-.15-.05c-4.39-1.45-8.79-3.07-13.45-4.8-8.23-3-16.74-6.18-26.58-8.87v-28.81a89.08 89.08 0 0 1 1.82-17.77 88.29 88.29 0 0 1 86.45-70.45H272a88.35 88.35 0 0 1 87.75 78.67 93.27 93.27 0 0 1 .53 9.6Z" transform="translate(-53.51 -32.37)" style="fill:#1bd8f4;stroke:#1bd8f4"/></svg>
```
Comment (wiz) [inline frontend/src/app/components/about/about.component.html:390]:
```suggestion
        <svg xmlns="http://www.w3.org/2000/svg" focusable="false" viewBox="0 0 440 440"><path d="M225.57,2.08l-.69-.45a4.22,4.22,0,0,0-5.72,1.23L182.33,46.09a4,4,0,0,0,.88,5.81l9.38,6.38L173.48,97.49a4.22,4.22,0,0,0,2.45,4.19s3.55.7,4.53-1l41.92-40.56a3.62,3.62,0,0,0-1.51-5.1l-10.55-6.12L227.44,6.79A4.26,4.26,0,0,0,225.57,2.08Z" fill="currentColor"></path><path d="M118.52,401.83c-62.51,0-113.37-51-113.37-113.67V214.68C5.15,152,56,101,118.52,101H342.08a24.82,24.82,0,0,1,24.76,24.83V377a24.81,24.81,0,0,1-24.76,24.82Z"></path><path d="M342.08,105.18a20.65,20.65,0,0,1,20.61,20.66V377a20.66,20.66,0,0,1-20.61,20.66H118.52C58.3,397.67,9.31,348.55,9.31,288.16V214.68c0-60.38,49-109.5,109.21-109.5H342.08m0-8.34H118.52C53.62,96.84,1,149.6,1,214.68v73.48C1,353.24,53.62,406,118.52,406H342.08A29,29,0,0,0,371,377V125.84a29,29,0,0,0-28.92-29Z" fill="currentColor"></path><path fill="currentColor" d="M344.69,346.23A25.84,25.84,0,1,0,335,369.87l-10.22-10.2a11.69,11.69,0,1,1,4.77-5.12l10.31,10.28A25.84,25.84,0,0,0,344.69,346.23Z"></path><path fill="currentColor" d="M315.82,257.61a25.67,25.67,0,0,0-12.53,5.22L315,274.49a9.58,9.58,0,0,1,2.11-.73A9.72,9.72,0,1,1,309.4,283a9.4,9.4,0,0,1,.75-3.41L298.4,267.84a25.77,25.77,0,1,0,17.42-10.23Z"></path><path fill="currentColor" d="M313,214a7.76,7.76,0,1,1,1.41,10.91,7.62,7.62,0,0,1-2.19-2.69l-18.67-.14a25.94,25.94,0,1,0,.05-7l18.64.14A7.4,7.4,0,0,1,313,214Z"></path><path fill="currentColor" d="M341.2,144.08h-6.32c-1.67,0-3.61,1.87-3.61,4.29s1.94,4.29,3.61,4.29h6.32c1.67,0,3.61-1.87,3.61-4.29S342.87,144.08,341.2,144.08Z"></path><path fill="currentColor" d="M301.75,144.08h-6.44c-1.67,0-3.61,1.87-3.61,4.29s1.94,4.29,3.61,4.29h6.44c1.67,0,3.61-1.87,3.61-4.29S303.42,144.08,301.75,144.08Z"></path><path fill="currentColor" d="M321.77,144.08h-7c-1.67,0-3.62,1.87-3.62,4.29s1.95,4.29,3.62,4.29h7c1.67,0,3.62-1.87,3.62-4.29S323.44,144.08,321.77,144.08Z"></path><ellipse fill="currentColor" cx="295.97" cy="127.61" rx="4.27" ry="4.29"></ellipse><path fill="currentColor" d="M340.54,131.9a4.29,4.29,0,1,0-4.27-4.29A4.28,4.28,0,0,0,340.54,131.9Z"></path><path fill="currentColor" d="M318.26,131.9a4.29,4.29,0,1,0-4.27-4.29A4.29,4.29,0,0,0,318.26,131.9Z"></path><ellipse fill="currentColor" cx="295.97" cy="169.13" rx="4.27" ry="4.29"></ellipse><path fill="currentColor" d="M340.54,164.84a4.3,4.3,0,1,0,4.27,4.29A4.29,4.29,0,0,0,340.54,164.84Z"></path><path fill="currentColor" d="M318.26,164.84a4.3,4.3,0,1,0,4.28,4.29A4.29,4.29,0,0,0,318.26,164.84Z"></path><path d="M108.62,256.87c8.36-1,7.68-7.76,3.14-17-3.64-7.4-9.74-16.39-15.75-25.36-14.23-21.23-27.69-42.23-5.35-41.07,19.55,1,42.9,18.63,68.22,36.74,31.1,22.24,65.16,45.21,98.81,39.11a151.19,151.19,0,0,1,20-2.37V221a92,92,0,0,0-91.91-92.16H124.33A92,92,0,0,0,32.42,221v17.59c17.71,3.81,31,9.94,43.8,14.15C86.6,256.16,96.69,258.31,108.62,256.87Z"></path><path d="M273.37,310.79c-35-15.26-76.67-32.1-104-23.59-3.15,1-5,2.3-6,3.85-3.35,5.31,4.67,13.57,14.89,22.17,7.17,6,15.36,12.21,21.44,17.64,11.47,10.26,15.35,17.84-9.89,16.62-29.75-1.44-49.18-13.75-71.18-24l-.29-.14a165.84,165.84,0,0,0-22.93-8.91c-15.74-4.67-34.22-6.79-58.51-3.28A91.93,91.93,0,0,0,124.33,375h61.45A92,92,0,0,0,273.37,310.79Z"></path><path fill="currentColor" d="M257.69,249.31C224,255.41,190,232.44,158.88,210.2c-25.32-18.11-48.67-35.72-68.22-36.74C68.32,172.3,81.78,193.3,96,214.53c6,9,12.11,18,15.75,25.36,4.54,9.22,5.22,16-3.14,17-11.93,1.44-22-.71-32.4-4.13-12.8-4.21-26.09-10.34-43.8-14.15v44.26c0,1.26.14,2.48.19,3.72a91.8,91.8,0,0,0,2.9,19.62c.43,1.67.84,3.34,1.37,5,24.29-3.51,42.77-1.39,58.51,3.28a165.84,165.84,0,0,1,22.93,8.91c.39-.12.76-.26,1.14-.39l-.85.53c22,10.25,41.43,22.56,71.18,24,25.24,1.22,21.36-6.36,9.89-16.62-6.08-5.43-14.27-11.61-21.44-17.64-10.22-8.6-18.24-16.86-14.89-22.17,1-1.55,2.87-2.87,6-3.85,27.33-8.51,69,8.33,104,23.59.32-1,.56-2.05.84-3.07a92.33,92.33,0,0,0,3.48-24.87V246.94A151.19,151.19,0,0,0,257.69,249.31Z"></path><path fill="currentColor" d="M192,137a78,78,0,0,1,77.78,78v73.91a78,78,0,0,1-77.78,78H118.51a78,78,0,0,1-77.78-78V215a78,78,0,0,1,77.78-78H192m0-8.33H118.51A86.21,86.21,0,0,0,32.42,215v73.91a86.21,86.21,0,0,0,86.09,86.33H192a86.21,86.21,0,0,0,86.09-86.33V215A86.21,86.21,0,0,0,192,128.64Z"></path></svg>
```

**Review by softsimon — APPROVED** [review]
tested ACK @ v2.5.0-dev [[19e2d687](https://github.com/mempool/mempool/commit/19e2d687)]

**Pull request merged**